htlcswitch/switch.go

```diff
@@ -621,7 +604,7 @@ func (s *Switch) ForwardPackets(linkQuit chan struct{},
 		default:
 			err := s.routeAsync(packet, fwdChan, linkQuit)
 			if err != nil {
-				return errChan
+				return fmt.Errorf("failed to forward packet %v", err)
```

Comment:
Hmm this is an error that previously disappeared?

Reply:
Yes but it only happened if the switch or the link has been shut down.